lib/auth/clt.go

```diff
@@ -129,7 +129,8 @@ func NewHTTPClient(cfg client.Config, tls *tls.Config, params ...roundtrip.Clien
 		if len(cfg.Addrs) == 0 {
 			return nil, trace.BadParameter("no addresses to dial")
 		}
-		contextDialer := client.NewDialer(cfg.KeepAlivePeriod, cfg.DialTimeout)
+		// Deliberately ignore HTTP proxies for backwards compatibility.
+		contextDialer := client.NewDirectDialer(cfg.KeepAlivePeriod, cfg.DialTimeout)
```

Comment:
Is this change related to the reverse tunnel issue? Won't this break the actual scenario in which tsh should be respecting the proxies?